crowbar_framework/app/views/backups/index.html.haml

```diff
@@ -0,0 +1,75 @@
+.row
+  .col-xs-12
+    %h2.page-header
+      = t(".title")
+      .btn-group.pull-right
+        .btn.btn-primary{ data: { toggle: "modal", target: "#create_backup_modal" } }
+          = t(".create_image")
+        .btn.btn-default{ data: { toggle: "modal", target: "#upload_backup_modal" } }
+          = t(".upload_image")
+.panel.panel-default
+  %table.table
+    %thead
+      %tr
+        %th.col-xs-3
+          = t(".image_name")
+        %th.col-xs-3
+          = t(".version")
+        %th.col-xs-3
+          = t(".created_at")
+        %th.col-xs-3
+          = t(".file_size")
+        %th.col-xs-1
+        %th.col-xs-1
+        %th.col-xs-1
+    %tbody
+      - @backups.each do |backup|
+        %tr
+          %td
+            = backup.name
+          %td
+            = backup.version
+          %td
+            = l(backup.created_at, format: :long)
+          %td
+            = number_to_human_size(backup.size)
+          %td
+            = download_backup_button(backup)
+          %td
+            = restore_backup_button(backup)
+          %td
+            = delete_backup_button(backup)
+
+.modal.fade{ id: "create_backup_modal" }
+  .modal-dialog
+    .modal-content
+      .modal-header
+        .btn.close{ data: { dismiss: "modal" } }
+          &times
+        %h2
+          = t(".create_image")
+      = form_for Backup.new do |f|
```

Comment:
Never call models directly in a view, at least you should do it within a helper.